internal/trigger/gaussian/gaussian_rate.go

```diff
@@ -15,11 +16,12 @@ import (
 
 func GaussianRate() api.Builder {
 	flags := pflag.NewFlagSet("gaussian", pflag.ContinueOnError)
-	flags.Float64("volume", 24*60*60, "The desired volume to be achieved with the calculated load profile.")
+	flags.Float64("volume", 0, "The desired volume to be achieved with the calculated load profile.")
 	flags.Duration("repeat", 24*time.Hour, "How often the cycle should repeat")
 	flags.Duration("iteration-frequency", 1*time.Second, "How frequently iterations should be started")
 	flags.String("weights", "", "Optional scaling factor to apply per repetition. This can be used for example with daily repetitions to set different weights per day of the week")
 	flags.Duration("peak", 14*time.Hour, "The offset within the repetition window when the load should reach its maximum. Default 14 hours (with 24 hour default repeat)")
+	flags.StringP("peak-rate", "r", "", "number of iterations per interval in peak time, in the form <request>/<duration>")
```

Comment:
maybe we would need more information, that this value is only used when the volume is not provided.